packages/vue-app/template/plugins.js

```diff
@@ -0,0 +1,124 @@
+import Vue from 'vue'
+<% if (features.hookablePlugins) {
+%>import Hookable from 'hable'
```

Comment:
Including hable adds more functionality then we use/need at the moment. Not sure if its worth it to use the full hable? We could either use our own copy or maybe hable could export a slim version?

Reply:
I think keeping hable is ok. 

Not sure it's worth maintaining a separate version just for saving a couple hundred bytes.

Reply:
We can do it easily if needed to introduce slim browser version. But what exactly you think is extra that worth trim for slim build?